requirements.txt

```diff
@@ -4,6 +4,8 @@ requests
 matplotlib
 numpy
 socceraction
-warnings
```

Comment:
https://stackoverflow.com/questions/47722353/how-to-install-warnings-package-in-python